### ISSUE TITLE
My-session: session is showing multiple times 

### ISSUE THREAD

**Issue by maze-runnar** [fossasia/open-event-frontend#3752]
**Describe the bug**
My-session: session is showing multiple times 

**Expected behaviour**
session should show only once

**Screenshots**
[![multiple session](http://img.youtube.com/vi/EMXLyCehYCM/0.jpg)](http://www.youtube.com/watch?v=EMXLyCehYCM "multiple session")

**Desktop (please complete the following information):**

**Smartphone (please complete the following information):**

**Additional context**
i want to work on this

**Comment by kushthedude** [comment]
GIF ?

**Comment by maze-runnar** [comment]
http://www.youtube.com/watch?v=EMXLyCehYCM

**Comment by kushthedude** [comment]
Only accepted and confirmed session should be shown in `My Session`.

**Comment by iamareebjamal** [comment]
This is a server issue. Please open it there as well. API should by default only return accepted and confirmed sessions

**Comment by kushthedude** [comment]
> This is a server issue. Please open it there as well. API should by default only return accepted and confirmed sessions

No I got it wrong, All sorts of sessions should be displayed under `My Sessions` whether it be Accepted, Rejected or any.

**Comment by maze-runnar** [comment]
But they are showing multiple times. That's what the issue was about .

**Comment by kushthedude** [comment]
I didnt close the issue .

**Comment by maze-runnar** [comment]
oh! so i should create  another PR for this.

**Comment by maze-runnar** [comment]
@kushthedude on removing the `infinity loader` the sessions are rendering one time. So should i remove it ?
 ```
{{#infinity-loader infinityModel=model}}
      <div class="ui loading very padded basic segment">
      </div>
      {{infintyModel.reachedInfinity}}
    {{/infinity-loader}}
```

**Comment by kushthedude** [comment]
No there is no bug in infinity model as we are using at various different
places.
There must be bug in its implementation if you are saying it fixes without
it.

**Comment by maze-runnar** [comment]
could there be any problem in model ?

**Comment by maze-runnar** [comment]
@kushthedude i am not getting this clearly , is it a server side issue . Because model is rendering sessions multiple times when used in `infinityModel=model` .
Can you please suggest me what to do :pray:

**Comment by kushthedude** [comment]
I didn't get time to investigate. The issue may or may not be due to the
infinityModel Lacking or some error in our implementation.
Can you look a further into it , as infinityModal has been serving quite
well in My Tickets,explore and index route.

**Comment by maze-runnar** [comment]
Yes , isee that infinityModal has been serving quite well in My Tickets that's why i have a doubt there could be a problem in model , perhaps.

**Comment by maze-runnar** [comment]
@kushthedude i tried that there could be a problem in session card so i remove that too but the actual problem is with model , it is returning every session twice .
![Screenshot from 2020-01-10 23-09-35](https://user-images.githubusercontent.com/56407566/72174881-bc2e0b80-3400-11ea-99aa-c36a75f7d613.png)
 and yes `infinity loader` is working fine.

**Comment by maze-runnar** [comment]
@uds5501 can you please help here .

**Comment by maze-runnar** [comment]
@kushthedude @uds5501 for the time can i use this 
https://stackoverflow.com/questions/23731119/iterate-over-json-object-duplicates-in-handlebars

**Comment by maze-runnar** [comment]
one more question , why to show unscheduled sessions in upcoming sessions or mysessions , can i remove them from that section , or arrage them on the basis of there event endTime ?

**Comment by maze-runnar** [comment]
@iamareebjamal it's a server side issue i think. That is returning every session twice. Please take a look here.

**Comment by iamareebjamal** [comment]
Paste the API link which is returning session twice

**Comment by maze-runnar** [comment]
http://open-event-api.herokuapp.com//v1/events/{event_identifier}/sessions{?sort,filter}
i think this, as list all events under the user and then list all sessions under that events.

**Comment by kushthedude** [comment]
The issue is due to the infinityModel. Reverting it until fixed.

**Comment by kushthedude** [comment]
Not a Frontend Issue
